GOVERNANCE.md

```diff
@@ -0,0 +1,63 @@
+# Project Governance
+*Last updated June 9, 2022*
+
+## Overview
+
+FOSSBilling is an open source project which builds and maintains a free, open source billing and client management solution. 
+
+Anybody with an interest can join the community, contribute to the project, and be a part of the decision making process. In the next few paragraphs we will explain how that process takes place. 
+
+__TLDR; Project decisions are made by a consensus of the projects steering committee and informed by community discussion.__
+
+## Community
+
+The FOSSBilling community is made up of individuals, companies, or groups who use and/or contribute to the FOSSBilling software.
+
+Anybody can use the core FOSSBilling software within the terms of the [license](https://github.com/FOSSBilling/FOSSBilling/blob/main/LICENSE). There are not and never will be any fees for usage, and no special requirements.
+
```

Comment:
Please contribute (contribution.md link)

And donate at open collective!

Reply:
Done